corehq/messaging/smsbackends/twilio/models.py

```diff
@@ -16,6 +16,7 @@
 # https://www.twilio.com/docs/api/errors/reference
 INVALID_TO_PHONE_NUMBER_ERROR_CODE = 21211
 WHATSAPP_LIMITATION_ERROR_CODE = 63032
+TO_FROM_BLACKLIST_ERROR = 21610
```

Comment:
nit: how about using the same naming convention ending with `_ERROR_CODE`?

Reply:
cf95d1f4374